lib/styles/settings/_z-indexes.scss

```diff
@@ -0,0 +1,3 @@
+// z-index positioning
+// hardcoded
+$z-index-modal: 16777270; // 16777271 is the highest possible value in Safari, we need to make space for elements that above modal, so -1
```

Comment:
It will bite us in the future, mark my words 🙂 
I have no good alternatives for a better, cheap solution, though.

Reply:
Do you want to share what's in your mind?

Reply:
Yes, we did, but now the same problems are distributed 😉  If we use this var in marketplace app then it'll be even harder to fix bugs. We don't have to solve it now.